src/Xrefcheck/Verify.hs

```diff
@@ -236,6 +239,18 @@ instance Given ColorMode => Buildable VerifyError where
       #{err}
       |]
 
+    PermanentRedirectError url Nothing ->
+      [int||
+      Permanent redirect found:
+      #{url}
+      |]
+
+    PermanentRedirectError url (Just redirectedUrl) ->
+      [int||
+      Permanent redirect found. Perhaps you want to update the link:
+      #{url} -> #{redirectedUrl}
```

Comment:
Perhaps I would put the links on separate lines, for two reasons:
1. They might be large;
2. This way they will be easier to compare visually. For instance, in case of `http` -> `https` suggestion, if the link is large then the user may struggle to see what is the difference.

Reply:
Right, this "from `http` to `https`" example is a clear case in which having the links reported in separate lines can be useful.